### PR TITLE
Error route added in app.js, routes/event.js, routes/admin.js , routes/index.js on backend branch 

### PR DESCRIPTION
**Pull request by guptapranav4299**
### Solves Issue #<number>
<Do not delete this template>
Description: <provide a description> The title should be short and relevant.

#### Checklist
- [ ] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [ ] I've tested my code on a Chromium based browser.
- [ ] I've tested my code on Mozilla Firefox.
- [ ] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
<provide a relevant screenshot or any testimonial for the solved issue if applicable>

### CODE REVIEW

**Comment by ankiiitraj** [comment]
Review might take a while as maintainers are busy!

**Review by EmperorYP7 — APPROVED** [review]
LGTM

**Comment by vishalvibes** [comment]
@EmperorYP7 @guptapranav4299 the checks have failed...is it expected behaviour?

**Comment by ankiiitraj** [comment]
This is ok @Vishal19111999

**Review by ankiiitraj — APPROVED** [review]

**Comment by EmperorYP7** [comment]
I think there's a problem here @ankiiitraj
image

File: backend/app.js [hunk 1]
```diff
@@ -43,4 +43,8 @@ app.use(function(err, req, res, next) {
   res.render('error');
 });
 
+app.get('*', function(req, res){
+  res.status(500).json('bad request or 404 not found');
+});
+
```
Comment (anktrj) [inline backend/app.js:49]: this might be the problem!
 fixing this

**Pull request merged**